### DISCUSSION TITLE
Feature request - replace undetected-ChromeDriver with ultrafunkamsterdam/nodriver

### DISCUSSION THREAD

**Discussion by johnkennyuk** [FlareSolverr/FlareSolverr#1096] [Ideas]
I recently discovered ultrafunkamsterdam's repository for nodriver there new and faster replacement of undetected-ChromeDriver, I am yet to play with it personally but from ita readme.md it sounds like it should be fairly simple to implement. Also if its claims to being faster I'm all for it, apparently no need for selenium.

Unfortunately I am not experienced enough to develop any modifications to flaresolverr myself but I will try as flaresolverr's present performance isn't the best because of using selenium with driver.

I would like to hear if this has already been planned, or if not hear thoughts on this Idea?

Thanks

**Comment by johnkennyuk** [comment]
My bad finially noticed nodriver is there, although not sure how far implentation has got seeing as selenium is still in use??

**Comment by ilike2burnthing** [comment]
Not ready yet - https://github.com/ultrafunkamsterdam/nodriver/issues/1
